docs/cli.html

```diff
@@ -693,7 +693,7 @@ <h2 id="init">init</h2>
 <p>Note: this doesn&apos;t register or deploy the app with Zapier - try <code>zapier register &quot;Example&quot;</code> and <code>zapier push</code> for that!</p>
 </blockquote><p><strong>Arguments</strong></p><ul>
 <li><code>path [value]</code> -- <strong>required</strong>,</li>
-<li><code>--template={minimal,resource,trigger,create,search,middleware,basic-auth,custom-auth,oauth2,session-auth,babel,rest-hooks,files}</code> -- <em>optional</em>, select a starting app template. Default is <code>minimal</code></li>
+<li><code>--template={minimal,resource,trigger,create,search,middleware,basic-auth,custom-auth,oauth2,session-auth,babel,rest-hooks}</code> -- <em>optional</em>, select a starting app template. Default is <code>minimal</code></li>
```

Comment:
I'm not sure why `files` was removed, but I'm sure it's rebased with `master`. I couldn't revert it back by running the command that generated it: `npm run docs`. I just manually added it back. What do you think @BrunoBernardino?